backend/orchestrator/tsconfig.json

```diff
@@ -3,7 +3,12 @@
   "compilerOptions": {
     "preserveSymlinks": true,
     "baseUrl": ".",
-    "esModuleInterop": true
+    "esModuleInterop": true,
+    "paths": {
+      "functions/*": ["functions/*"],
+      "libs/*": ["libs/*"],
+      "resources/*": ["resources/*"]
+    }
```

Comment:
It is already infered from `baseUrl`, we do not neet to explicitely add the `paths` here